sys/shell/commands/sc_netif.c

```diff
@@ -36,6 +36,7 @@
 #include "net/gnrc/pktbuf.h"
 #include "net/gnrc/netif/hdr.h"
 #include "net/gnrc/sixlowpan/netif.h"
+#include "net/netdev2/ieee802154.h"
```

Comment:
>What if the interface isn't an IEEE 802.15.4 device?
>Are there devices that have different CCA modes, but are not IEEE 802.15.4? 

Then e.g. ``#include "net/netdev2/ieee80211x.h"``, yes some modes are different.

I tend to show/configure the wireless-specific stuff over own command like iwconfig or iwpan:

```
iwpan
    ....
    TX-Power: 0dBm  State: RX  max. Retrans.: 0  CSMA Retries: 0 
    AUTOACK  CSMA  CCA-MODE ED ......
```